src/isIsogram.test.js

```diff
@@ -6,4 +6,40 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return boolean value', () => {
+    const result = isIsogram('yes');
+
+    expect(typeof result).toBe('boolean');
+  });
+
+  it('should return true for an empty string', () => {
+    const result = isIsogram('');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return true if there are no repeated letters', () => {
+    const result = isIsogram('friends');
+
+    expect(result).toBe(true);
+  });
+
+  it('should return false if the consecutive letters repeat', () => {
+    const result = isIsogram('sunny');
+
+    expect(result).toBe(false);
+  });
+
+  it('should return false if the non-consecutive letters repeat', () => {
+    const result = isIsogram('without');
+
+    expect(result).toBe(false);
```

Comment:
The test case for non-consecutive letter repetition is incorrect. The word 'without' does not have repeating letters, so it is an isogram. The expected result should be `true`, not `false`.